content/reference-rippled.md

```diff
@@ -2177,7 +2175,7 @@ An example of the request format:
 *Commandline*
 
 ```
-#Syntax account_tx account [ledger_index_min [ledger_index_max [limit]]] [binary] [count] [forward]
+#Syntax account_tx account [ledger_index_min [ledger_index_max [limit]]] [binary] [forward]
```

Comment:
I think the commandline syntax still has the `count` parameter, and because the arguments are positional, if you leave it off, it'll interpret the next value as "count" even if you meant to to be "forward". So you may have to revert this line.

Reply:
Thank you! I see your point. All I did was edit a comment - not the actual command. The second "false" represents the count command -- if it still exists for command line. Added count back. 

@nbougalis - any thoughts on Rome's comment here? Does the command line syntax still include the count parameter?